docs/designs/topology.md

```diff
@@ -83,18 +92,20 @@ while allowing users to specify required constraints for strict placement (upper
 #### ClusterTopology CR
 
 ClusterTopology is a cluster-scoped CR that defines consistent naming for cluster topology hierarchy to be used by
-workload designers. It maps topology level domains to Kubernetes node labels and establishes ordering from broadest to
-narrowest scope.
+workload designers. It maps topology level domains to Kubernetes node labels.
 
 **Characteristics:**
 
-- **Cluster-scoped resource**: Can have multiple ClusterTopology resources defined
-- **Default name**: In operator configuration, clusterTopologyName defaults to "grove-topology" when not specified
-- **Partially immutable**: After creation, only `key` field values can be updated; `domain` fields and level ordering
-  are immutable
-- **List-ordered hierarchy**: Index 0 represents the broadest category (e.g., region), and the final index represents the narrowest (e.g., host).
-- **Supported topology levels**: Region > Zone > DataCenter > Block > Rack > Host > Numa (broadest to narrowest)
-- **Webhook-validated**: Webhook validates constraint hierarchy and immutability
+- **Cluster-scoped resource**: Only one ClusterTopology resource managed by operator: "grove-topology"
+- **Operator-managed resource**: Created and managed by Grove operator based on OperatorConfiguration
+- **User-created CRs allowed**: Users can create additional ClusterTopology CRs but Grove does not use them
```

Comment:
Ok, but is there a reason why we allow it? Seems like just preventing it might be safer?